src/platform/src/main/java/org/geoserver/platform/resource/Paths.java

```diff
@@ -127,7 +161,11 @@ private static String toPath(List<String> names) {
                 throw new IllegalArgumentException("Contains invalid " + item + " path: " + buf);
             }
             if (!WARN.matcher(item).matches()) {
-                // to do warning 
+                if (STRICT_PATH == true) {
+                    throw new IllegalArgumentException("Contains invalid " + item + " path: " + buf);
+                } else {
+                    // TODO: warning
```

Comment:
Do we still want to provide the front page UI warning though? Ignoring here sounds fine if we plan to do that.